volatility3/framework/plugins/windows/svcscan.py

```diff
@@ -140,18 +144,21 @@ def create_service_table(
             config_path,
             os.path.join("windows", "services"),
             symbol_filename,
-            class_types=services.class_types,
+            class_types=services_types.class_types,
             native_types=native_types,
         )
 
-    def _get_service_key(self, kernel) -> Optional[objects.StructType]:
+    @classmethod
+    def _get_service_key(
```

Comment:
If this one is now exposed, we should really remove the `_` from the front of it (so that we don't assume it's part of the private interface, but that changes need version bumps).

Reply:
I fixed this. That method should have been static and the create service table should have _. All those methods called by get_prereq_info should only be called by get_prereq_info so that the prerequisite data is built correctly. If i need to bump versions let me know, but svcscan already has a major version bump in this branch: https://github.com/volatilityfoundation/volatility3/pull/1168/files#diff-430aa6cb7e2040f3039b1de6da0e184992788d6f2cc1795af6f5f171fb774717R42